litellm/llms/custom_httpx/http_handler.py

```diff
@@ -32,15 +32,19 @@ class AsyncHTTPHandler:
     def __init__(
         self,
         timeout: Optional[Union[float, httpx.Timeout]] = None,
+        event_hooks: Optional[Mapping[str, list[Callable[..., Any]]]] = None,
         concurrent_limit=1000,
     ):
         self.timeout = timeout
         self.client = self.create_client(
-            timeout=timeout, concurrent_limit=concurrent_limit
+            timeout=timeout, concurrent_limit=concurrent_limit, event_hooks=event_hooks
```

Comment:
can you add a unit test here - https://github.com/BerriAI/litellm/blob/5e381caf757556a927c24f3085d3dc14e3be3a0b/tests/local_testing/test_utils.py

to make sure this works as expected

Reply:
Done. Let me know if the test looks good to you.